### PR TITLE
Use published modules from ACR

### PR DESCRIPTION
**Pull request by vhvb1989**
Published all /core bicep modules to: br:publicazdbicepmodules.azurecr.io
Updated infra modules to consume core modules from ACR
Removed /core bicep files from templates

### CODE REVIEW

File: templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep [hunk 1]
```diff
@@ -10,7 +10,7 @@ param appSettings object = {}
 param keyVaultName string
```
Comment (jongio) [inline templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep:10]: Please include a bicepconfig.json file like we have here: https://github.com/Azure/azure-dev/pull/1251/files

File: templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep [hunk 2]
```diff
@@ -10,7 +10,7 @@ param appSettings object = {}
...
 param serviceName string = 'api'
 
-module api '../../../../../common/infra/bicep/core/host/appservice.bicep' = {
+module api 'br:publicazdbicepmodules.azurecr.io/bicep/staging/core/host/appservice:v1.0' = {
```
Comment (jongio) [inline templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep:13]: I think we should move to semver

File: templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep (hunk 1 above)
Comment (vhvb1989) [reply templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep:10]: done

File: templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep (hunk 2 above)
Comment (vhvb1989) [reply templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep]: The format for the tag is not really relevant.  We can change it at any time, for any repository.
There won't be anything on the registry to validate the tag is `semver`.
For the production registry, we can limit the access for publishing modules to some app which would integrate the `semver` validation, but this is orthogonal to this PR.

Comment (jongio) [reply templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep]: It is critical that we figure out how we version modules and push those versions out to new templates.  I don't mind if you do it in a different PR, but figuring that out is a P1 to ship this.

**Pull request closed without merging**